docs/nachet-inference-documentation.md

```diff
@@ -1,12 +1,24 @@
 # Inference Request with n pipelines
 
+## Executive summary
+Following the objective of milestones 1, Nachet back end need to deploy a
+functionality to switch in between different model. However, a model can call
+another model for more accurate result. Therefore creating a pipeline of models.
+In order to not be mix in between terms, a glossary to explain what is a
+pipeline and what is model was needed.
+
+Since pipeline can have n models, the use of the blob storage as a temporarily
```

Comment:
implementation details, not in executive summary

Reply:
Done, change executive summary and try to implement your reviews.